src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/PropertySetAnalysis/PropertySetAnalysisContext.cs

```diff
@@ -156,12 +156,12 @@ internal sealed class PropertySetAnalysisContext : AbstractDataFlowAnalysisConte
         public ImmutableDictionary<(INamedTypeSymbol, bool), string> HazardousUsageTypesToNames { get; }
 
 #pragma warning disable CA1307 // Specify StringComparison - string.GetHashCode(StringComparison) not available in all projects that reference this shared project
-        protected override void ComputeHashCodePartsSpecific(Action<int> addPart)
+        protected override void ComputeHashCodePartsSpecific(Action<int> builder)
```

Comment:
Honestly `addPart` seems a better name than `builder` for a delegate, but I don't have strong opinion either way.

Reply:
> Honestly `addPart` seems a better name than `builder` for a delegate, but I don't have strong opinion either way.

Will update in the name in base class to `addPart`.